frontend/src/metabase/query_builder/actions/core/parameterUtils.ts

```diff
@@ -86,7 +86,7 @@ export function getParameterValuesForQuestion({
   metadata: Metadata;
 }) {
   const parameters = getCardUiParameters(card, metadata);
-  return getParameterValuesByIdFromQueryParams(parameters, queryParams);
+  return getParameterValuesByIdFromQueryParams(parameters, queryParams ?? {});
```

Comment:
nit: `queryParams` type should be changed to `queryParams: QueryParams` and in the place of usage it will be

```
getParameterValuesForQuestion({
    card: finalCard,
    queryParams: queryParams ?? {},
    metadata,
  });
```